app/runner.go

```diff
@@ -56,7 +56,11 @@ func (m *MultiRunner) Run(ctx context.Context) error {
 			err := r.Run(propagatedContext)
 			wg.Done()
 			if err != nil && !timedOut {
-				errs <- err
+				select {
```

Comment:
THis is a drive-by fix for an edge case that crops up every so often in tests where the channel is already closed because the runner has exited, but the goroutine hadn't finished yet.